app/amity.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
```

Comment:
I'm not too clear on the purpose of this file. They appear to have similar functionality of running the docopt interface.
either remove one file or combine all working docopt functionality into one file.